lib/domain/app_state/room/create_room_success.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:fluffychat/app_state/success.dart';
+
+enum CreateRoomSuccessEventType {
+  createRoomSuccess,
+  setAvatarSuccess,
+}
+
+class CreateRoomSuccess extends Success {
+  final CreateRoomSuccessEventType event;
+  final String? roomId;
+
+  const CreateRoomSuccess({required this.event, required this.roomId});

```

Comment:
`required  this.roomId` => `this.roomId`

Reply:
Why ? Every success (createRoom with / without avatar) have necessarily a Room ID

Reply:
Yeah, but roomId is nullable. if so make `final String? roomId;` => `final String roomId;`

Reply:
that the problem, some `success` need the `roomId`, some not. Please separate the instance of `success`.